index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ferizad's portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="homepage">
+        <header>
+            <h1 class="logo">Welcome</h1>
+            <img class="hamburger" src="media\Union.svg" alt="hamburger" width="20px" height="20px">
+        </header>
+        <section class="frontpage">
+            
+            <picture>
+                <source media="(max-width:465px)" srcset="media\header-illsutration-mobile@2x.png">
+                <img class="background" src="media\background.png" alt="Flowers">
+              </picture>
```

Comment:
- In order to add the background image for this section, please, kindly use the CSS background property to achieve this. Please, kindly remove the `<picture></picture>` and it contents totally from the document